assets/src/edit-story/app/story/useStoryReducer/actions.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Internal dependencies
+ */
+import * as types from './types';
+
+const addPage = ( dispatch ) => ( { properties } ) =>
+	dispatch( { type: types.ADD_PAGE, payload: { properties } } );
+
+const deletePage = ( dispatch ) => ( { pageIndex } ) =>
```

Comment:
I considered if I should switch to ID's half-way through (because pages have ID's too), but was too lazy. Damn, you caught me! Will update to ID's for everything related to page pointers.